### PR TITLE
devops: Try enabling binary install of `binstall`

### PR DESCRIPTION
**Pull request by max-sixty**
Potentially solving #3420

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -118,13 +118,9 @@ tasks:
...
     cmds:
-      # Currently pinning 1.1.2 because of https://github.com/cargo-bins/cargo-binstall/issues/1292, and using `+stable` because it's not supported by 1.68.2.
       - cmd:
-          # curl -L --proto '=https' --tlsv1.2 -sSf
-          # https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh
-          # | bash ||
-          rustup toolchain add stable && cargo +stable install --locked
-          cargo-binstall --version=1.1.2
+          curl -L --proto '=https' --tlsv1.2 -sSf
+          https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh
```
Comment (max-sixty) [inline Taskfile.yml:123]:
```suggestion
          https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh
          | bash
```

**Review by eitsupi — CHANGES_REQUESTED** [review]
Thanks, could you revert 28b9653b4d0a52427dde2269d2d5826d09a695f3?

**Comment by max-sixty** [comment]
> Thanks, could you revert [28b9653](https://github.com/PRQL/prql/commit/28b9653b4d0a52427dde2269d2d5826d09a695f3)?

Yes if this is fast!

Can we not block this from merging though?

**Comment by max-sixty** [comment]
@eitsupi should we merge? Or is this superseded by other PRs?

**Comment by eitsupi** [comment]
Yes! Closing in favor of #3423

**Pull request closed without merging**